start_docker.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+# define the version number, this needs to be updated every new Docker release
+VERSION='v0.4.2'
```

Comment:
Will you add this to the developer docs, there's a list of what needs to be changed for each release. Also see if anything else is missing from there